src/app/modules/main/chat_section/chat_content/messages/module.nim

```diff
@@ -256,13 +256,14 @@ method newMessagesLoaded*(self: Module, messages: seq[MessageDto], reactions: se
     self.view.model().removeItem(CHAT_IDENTIFIER_MESSAGE_ID)
     # Add new loaded messages
     self.view.model().appendItems(viewItems)
+    self.view.model().resetNewMessagesMarker()
+
+    # check if this loading was caused by the click on a messages from the app search result
+    self.checkIfMessageLoadedAndScrollToItIfItIs()
 
   if(not self.view.getInitialMessagesLoaded()):
```

Comment:
I just realized, this `if` is useless, since the view does the same check. Also, it doesn't matter much if we set it to `true` twice.